Workspaces/Base/Workspace.cs

```diff
@@ -142,12 +148,27 @@ public virtual void Setup()
 			m_WorkspaceUI.leftRayOrigin = leftRayOrigin;
 			m_WorkspaceUI.rightRayOrigin = rightRayOrigin;
 
-			m_WorkspaceUI.resize += bounds => { contentBounds = bounds; };
+			m_WorkspaceUI.resize += bounds =>
+			{
+				var size = contentBounds.size;
+				var boundsSize = bounds.size;
+				size.x = boundsSize.x;
+				size.z = boundsSize.z;
+				var content = contentBounds;
+				content.size = size;
+				contentBounds = content;
+			};
 
 			m_WorkspaceUI.sceneContainer.transform.localPosition = Vector3.zero;
 
+			m_OuterCollider = gameObject.AddComponent<BoxCollider>();
+			m_OuterCollider.isTrigger = true;
+			m_FrameCollider = gameObject.AddComponent<BoxCollider>();
+			m_FrameCollider.isTrigger = true;
+
+			var startingBounds = m_CustomStartingBounds ?? DefaultBounds;
 			//Do not set bounds directly, in case OnBoundsChanged requires Setup override to complete
-			m_ContentBounds = new Bounds(Vector3.up * DefaultBounds.y * 0.5f, m_CustomStartingBounds ?? DefaultBounds); // If custom bounds have been set, use them as the initial bounds
+			m_ContentBounds = new Bounds(Vector3.up * startingBounds.y * 0.5f, startingBounds); // If custom bounds have been set, use them as the initial bounds
```

Comment:
Yes. The transform of the workspace has its y = 0 point at the top of the frame. The content extends out from there, so as it grows in height, the center goes up by half the height.